py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/config_rt_alerts.py

```diff
@@ -1,19 +1,14 @@
 import pyarrow
 
-from .config_base import ConfigDetail
-from .config_base import ConfigType
+from .gtfs_rt_detail import GTFSRTDetail
 
 
-class RtAlertsDetail(ConfigDetail):
+class RtAlertsDetail(GTFSRTDetail):
     """
     Detail for how to convert RT GTFS Alerts from json entries into parquet
     tables.
     """
 
-    @property
-    def config_type(self) -> ConfigType:
-        return ConfigType.RT_ALERTS
-
```

Comment:
This is going to be the same change for all of the specialization GTFS RT files.

* Inherit from `GTFSRTDetail` instead of `ConfigDetail`
* no longer need to supply config type. `Converter` class has that.